test/gosec.spawn.test.js

```diff
@@ -1,36 +1,41 @@
 // Copyright 2018 VMware, Inc.
 // SPDX-License-Identifier: BSD-2-Clause
 
-const gosec = require('../gosec/gosec')
 const fs = require('fs-extra')
 
-describe('Spawn gosec tests', () => {
-  test('Empty input for gosec', async () => {
-    const gosecResult = await gosec('', [])
-    expect(gosecResult).toBeFalsy()
-  })
+const { run } = require('../runner')
+const Gosec = require('../linters/gosec')
+
+function gosec (dir, files) {
+  return run(new Gosec(), dir, files)
+}
+
+describe('Gosec runner', () => {
+  test('Finds issues in invalid file', async () => {
+    const report = await gosec('test/fixtures/go/src/bad_files', ['bad_test_file.go'])
 
-  test('Run gosec on non go files', async () => {
-    const gosecResult = await gosec('test/fixtures/go/src/non_go_files', ['hello_world.py'])
-    expect(gosecResult).toBeFalsy()
```

Comment:
I think that it's a good idea to keep that test were we give a non go file to gosec.
Because whatever changes we are doing we should be aware we are filtering for non go files.
The problem is that Gosec is really sensitive and when you give Gosec non go files it throws an exception.

Reply:
I'm not sure there is much we can do about this as we don't specify individual go files to scan, we just give it a directory and let gosec scan it recursively.
The logic to save go files to a specific directory is not part of the runner code so it doesn't make sense to test it here.

Reply:
I don't think we should download the go files in other special folder.
I think we should keep the structure of the project and pr.
We know if there are any go file for scan because we filter them and do that check:
![image](https://user-images.githubusercontent.com/16246778/51034530-bca68300-15af-11e9-8ce1-110429ffe858.png)

Reply:
Well we are doing this because the go code needs to be in the GOPATH which is currently set to cache/go.